### PR TITLE
Move to maven and GitHub actions

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+env:
+  BRANCH_NAME: version-bump-${{ github.run_id }}${{ github.run_attempt }}
+jobs:
+  deploy-db-changes:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: password
```
Comment (TomJKing) [inline .github/workflows/publish.yml:13]: Should the password and user values be hardcoded?

File: .github/workflows/run.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        run: print(f"::set-output name=role-name::TDRGithubActionsDeployLambda{'${{ github.event.inputs.environment }}'.title()}")
+        shell: python
+      - name: Configure AWS credentials for Lambda
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.ACCOUNT_NUMBER }}:role/${{ steps.role-name.outputs.role-name }}
+          aws-region: eu-west-2
+          role-session-name: ECRLogin
+      - name: Invoke lambda
+        run: aws lambda invoke --function-name tdr-database-migrations-${{ github.event.inputs.environment }} out
```
Comment (TomJKing) [inline .github/workflows/run.yml:28]: Missing line break

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor comments. Thanks

File: .github/workflows/publish.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/publish.yml:13]: They can be. It's just a local database that gets destroyed once the job finishes. We run `flywayMigrate` to populate the tables and stuff and then `slickCodegen` generates the classes from the database. The username and password are already defined [in build.sbt](https://github.com/nationalarchives/tdr-consignment-api-data/blob/master/build.sbt#L47) and we've got it hardcoded now in the [Jenkins node Dockerfile](https://github.com/nationalarchives/tdr-jenkins/blob/master/docker/nodes/postgres/Dockerfile#L10)

Comment (TomJKing) [reply .github/workflows/publish.yml:13]: Fair enough

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**